### ISSUE TITLE
Cannot scroll or select on intellisense with mouse when it's on top of the terminal when accessibility mode is enabled

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#76271]
I can only select the top 4 items with the mouse:

<img width="523" alt="Screen Shot 2019-06-27 at 6 43 29 PM" src="https://user-images.githubusercontent.com/2193314/60311311-7baaec80-990b-11e9-9b4b-a9797d95bbf0.png">

**Comment by jrieken** [comment]
Hm, this works for me...

![Jun-28-2019 06-51-12](https://user-images.githubusercontent.com/1794099/60317875-2d303900-9971-11e9-8557-67b2c02fffdc.gif)

**Comment by Tyriar** [comment]
Fixed the title, you need this for it to happen:

```
"editor.accessibilitySupport": "on"
```

**Comment by Tyriar** [comment]
Looks like it's to do with this style

```css
.xterm .xterm-accessibility, .xterm .xterm-message {
    position: absolute;
    left: 0;
    top: 0;
    bottom: 0;
    right: 0;
    z-index: 100;
    color: transparent;
}
```

I can lower that in the terminal, sorry about the ping 😅 https://github.com/xtermjs/xterm.js/issues/2275
